outwatch/src/main/scala/outwatch/dom/helpers/DomUtils.scala

```diff
@@ -222,7 +223,7 @@ private[outwatch] object NativeModifiers {
           lengthBefore += lengthsArr(i) getOrElse 1
           i += 1
         }
-        modifiers.splice(lengthBefore, lengthsArr(index) getOrElse 1, mods.toSeq: _*)
+        modifiers.asInstanceOf[js.Dynamic].splice.asInstanceOf[FunctionRawApply].applyCall(modifiers, js.Array(lengthBefore, lengthsArr(index) getOrElse 1).concat(mods))
```

Comment:
This call might look weird on the first glance, but was only way I was able to properly call `splice` on an Array. Normally `splice` takes a `VarArg*` for the value that should be inserted. Now with defensive copying and toSeq, we always get boilerplate code:

```